### PR TITLE
Add support for LTC2672

### PR DESCRIPTION
**Pull request by SaikiranGudla**
# Description

Add ltc2672.py driver
Add test script, example code
Update index.rst, init.py, supported_parts.md files Add ltc2672 rst file

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How has this been tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Hardware:
* OS:

# Documentation

If this is a new feature or example please mention or link any documentation. All new hardware interface classes require documentation.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have signed off all commits and they contain "Signed-off by: <insert name>"
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by SaikiranGudla** [comment]
Hi @tfcollins ,
This PR is ready to be reviewed.

File: adi/ltc2672.py [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+
+from adi.attribute import attribute
+from adi.context_manager import context_manager
+
+
+class ltc2672(context_manager, attribute):
+    """LTC2672 DAC"""
+
+    _complex_data = False
+    channel = []
```
Comment (tfcollins) [inline adi/ltc2672.py:15]: **channel** should not be a class property or it needs to be reset within the constructor. This creates an issue when you have multiple instances of the class. channel will keep growing

File: examples/ltc2672_example.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+# PARTICULAR PURPOSE ARE DISCLAIMED.
+#
+# IN NO EVENT SHALL ANALOG DEVICES BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, INTELLECTUAL PROPERTY
+# RIGHTS, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR
+# BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+# STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF
+# THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+from adi.ltc2672 import *
```
Comment (tfcollins) [inline examples/ltc2672_example.py:34]: Please do not use * imports. Just import what you need

**Review by tfcollins — CHANGES_REQUESTED** [review]
Just a few minor changes

**Comment by SaikiranGudla** [comment]
Change log: Addressed review comments

Import ltc2672 class in example file
Make channel an instance attribute instead of class attribute

**Pull request merged**